### PR TITLE
Add mechanism to display textual holdings (866 MARC field)

### PR DESCRIPTION
**Pull request by alexklbuckley**
Sponsored-By: Aucklan University of Technology, New Zealand

### CODE REVIEW

File: module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/RecordDataFormatterTest.php [hunk 1]
```diff
@@ -521,9 +521,9 @@ public function testFormatting(string $function): void
             'Language' => 'ItalianLatin',
             'Published' => 'Centro di Studi Vichiani, 1992',
             'Edition' => 'Fictional edition.',
-            'Series' => 'Vico, Giambattista, 1668-1744. Works. 1982 ;',
             'Multi Count' => 1,
             'Multi Data' => 'Book',
+            'Series' => 'Vico, Giambattista, 1668-1744. Works. 1982 ;',
```
Comment (demiankatz) [inline module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/RecordDataFormatterTest.php]: Is this change related to the rest of the PR? It certainly doesn't hurt to put the expectations in alphabetical order, but I don't see a reason why your other code changes would require this test to be updated. Did you start to work on adding a test case for the new functionality but decide not to proceed with it?

File: module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatterFactory.php [hunk 2]
```diff
@@ -333,6 +333,10 @@ public function getDefaultCoreSpecs()
                 'itemSuffix' => '</span>',
             ]
         );
+        $spec->setLine(
```
Comment (demiankatz) [inline module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatterFactory.php]: I wonder if it would be safer to make this an example in RecordDataFormatter.ini instead of turning it on by default. Since 866 is technically part of MARC holdings, it's not really a standards-compliant thing to do by default. I realize this is a common use case because of merging holdings records into bib records, but I imagine turning it on automatically might have unwanted/unexpected side effects for many people. Making it easy to activate while leaving it off to begin with seems like the safest path forward (though I'm open to discussing further if you or others feel differently).

**Review by demiankatz — CHANGES_REQUESTED** [review]
See below for a couple of initial thoughts...

File: module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatterFactory.php (hunk 2 above)
Comment (alexklbuckley) [reply module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatterFactory.php]: thanks @demiankatz . Yes, that makes perfect sense. In my latest follow-up I've shifted this over to be an example in the RecordDataFormatter.ini file. As I am not very familiar with that file I am not sure if I have added it correctly. If you wouldn;t mind checking?

File: module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/RecordDataFormatterTest.php (hunk 1 above)
Comment (alexklbuckley) [reply module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/RecordDataFormatterTest.php]: thanks @demiankatz yes this was due to me starting to work on adding a test case for the new functionality and then I decided not to proceed with it. I've removed this change in my latest follow up

**Review by demiankatz — APPROVED** [review]
Thanks, @alexklbuckley! Your example works for me, though I made a couple of minor adjustments -- changing "Holdings" to "TextualHoldings" to be more specific, and adding `pos = 2000` so the new field displays at the end of the record instead of the beginning.

**Comment by alexklbuckley** [comment]
thanks so much @demiankatz ! Appreciate all your help as always!
